.github/workflows/rust.yml

```diff
@@ -21,8 +21,8 @@ jobs:
         cargo clippy --workspace --all-targets -- -D warnings
     - name: Test Native Intrinsics
       run: |
-        cargo test
-        cargo test --release
+        cargo test --features serde
+        cargo test --features serde --release
```

Comment:
In general, it would be good to test all features (so `scale` as well) and not just `serde`.

Reply:
Replaced with `cargo hack`